x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('EmptyEngineOverview', () => {
 
   it('renders correctly versioned documentation URLs', () => {
```

Comment:
Ah do you mind changing this test name as well while you're here, sorry, since we're not manually rendering documentation versions anymore

```suggestion
  it('renders a documentation link', () => {
```